packages/ndla-code/src/Codeblock/Codeblock.tsx

```diff
@@ -47,6 +47,18 @@ const Wrapper = styled.div`
   }
 `;
 
+const TitleBar = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const ButtonBar = styled.div`
```

Comment:
trenger du denne? syns det ser litt rart ut som bruker når jeg trykker på knappen.

<img width="93" alt="Screenshot 2020-10-15 at 14 21 09" src="https://user-images.githubusercontent.com/10741877/96122406-badec180-0ef1-11eb-82e3-b6fdc2c056ab.png">

Reply:
Nei, det heng igjen etter at det var to knapper der.

Reply:
Fjerna denne.